All_years/04-Social_Care/04-SDS_functions.R

```diff
@@ -0,0 +1,70 @@
+#' Open a connection to a PHS database
```

Comment:
I thought I deleted this file on a previous commit. I think it can be deleted now anyway.

Reply:
Yeah this can be deleted, i think this is the same on A-T i noticed this still appeared earlier